src/Debugging-Core/InstructionStream.extension.st

```diff
@@ -24,12 +24,17 @@ InstructionStream >> debuggerMap [
 InstructionStream >> firstByte [
 	"Answer the first byte of the current bytecode."
 
-	^self method at: pc
+	^self method at: self method initialPC
```

Comment:
This new implementation does not go well with the comment. The comment is supposed to return the first byte of the **current bytecode**. The new implementation returns the first byte of the method.

Reply:
agreed!